visualization/resources/web/vis/genericChart/genericChartHelper.js

```diff
@@ -130,7 +130,9 @@ LABKEY.vis.GenericChartHelper = new function(){
                     ? properties[0].aggregate : properties.aggregate;
 
             if (LABKEY.Utils.isDefined(aggregateProps)) {
-                var aggLabel = LABKEY.Utils.isObject(aggregateProps) ? aggregateProps.name : LABKEY.Utils.capitalize(aggregateProps.toLowerCase());
+                var aggLabel = LABKEY.Utils.isObject(aggregateProps)
+                        ? (aggregateProps.name != null ? aggregateProps.name : aggregateProps.label)

```

Comment:
I think you can use `??` here and avoid a nested ternary.
```suggestion
                        ? (aggregateProps.name ?? aggregateProps.label)
```

Reply:
done